.github/actions/build-artifacts/action.yml

```
@@ -0,0 +1,40 @@
+name: Artifacts
+description: Build UI Kit Documentation and App
+
+inputs:
+  build-app: 
+    description: 'Whether the app should be built or not.'
+    required: false
+    type: boolean
+    default: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install Dependencies
+      uses: ./.github/actions/install-dependencies
+
+    - name: Build Documentation
+      run: npx cross-env NODE_OPTIONS=--max_old_space_size=8192 build-storybook --docs -o dist --quiet
```

Shouldn't this script live in our `package.json` scripts? We already have `build:doc`
`max_old_space_size` seems to be no longer necessary (was an old workaround)

```suggestion
      run: npm run build:doc
```

---

I believe we still need this `max_old_space_size` property. I tried using `npm run build:doc` and got the following [error](https://github.com/lumada-design/hv-uikit-react/actions/runs/5177698255/jobs/9328459698):

```
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
```